build_defs/cgo.build_defs

```diff
@@ -0,0 +1,140 @@
+subinclude("///go//build_defs:go", "///cc//build_defs:c")
```

Comment:
do you not need to define the `cc` plugin in this repo for this subinclude to work? or not getting that far because of this other bug?

Reply:
Ah yeah, I stopped after I realised I need to fix something on the Please side. Will get back to this.